Enabled landscape mode in the email application.
Can be toggled by typing 'wide' in the accounts-assistant

From: http://predev.wikidot.com/enable-landscape-viewing
jblebrun, xorg, FXDemolisher, andrewboudreau
Index: /usr/palm/applications/com.palm.app.email/app/controllers/accounts-assistant.js
===================================================================
--- .orig/usr/palm/applications/com.palm.app.email/app/controllers/accounts-assistant.js
+++ /usr/palm/applications/com.palm.app.email/app/controllers/accounts-assistant.js
@@ -51,6 +51,10 @@ var AccountsAssistant = Class.create(App
 		EmailAppDepot.depot.isShowFavorites(AccountsAssistant.kFavoriteFoldersAccountID, this.depotGetFavExpanded.bind(this));
 
 		this.controller.listen(this.controller.sceneElement, Mojo.Event.keypress, this.keypressHandler.bind(this));
+		
+		// enable free orientation
+		this.controller.window.PalmSystem.setWindowOrientation("free");
+
 	},
 
 	cleanup: function() {
@@ -102,8 +106,8 @@ var AccountsAssistant = Class.create(App
 	keypressHandler: function(e) {
 		this.orientationString += String.fromCharCode(e.originalEvent.charCode);
 		Mojo.Log.info(this.orientationString);
-		if (this.orientationString.length === 12) {
-			if (this.orientationString === "RocknRollHax") {
+		if (this.orientationString.length === 4) {
+			if (this.orientationString === "wide") {
 				Mojo.Log.info("Let's rock and roll");
 				var targetWindow = this.controller.window;
 				if (targetWindow.PalmSystem.setWindowOrientation) {
Index: /usr/palm/applications/com.palm.app.email/app/controllers/compose-assistant.js
===================================================================
--- .orig/usr/palm/applications/com.palm.app.email/app/controllers/compose-assistant.js
+++ /usr/palm/applications/com.palm.app.email/app/controllers/compose-assistant.js
@@ -178,6 +178,10 @@ var ComposeAssistant = Class.create({
 
 		// Delayed a little, since we want the header part of render as quickly as possible
 		ComposeAssistant.onLoad.defer(this.controller, this.email);
+		
+		// enable free orientation
+		this.controller.window.PalmSystem.setWindowOrientation("free");
+
 	},
 
 	cleanup: function() {
Index: /usr/palm/applications/com.palm.app.email/app/controllers/list-assistant.js
===================================================================
--- .orig/usr/palm/applications/com.palm.app.email/app/controllers/list-assistant.js
+++ /usr/palm/applications/com.palm.app.email/app/controllers/list-assistant.js
@@ -27,6 +27,10 @@ var ListAssistant = Class.create(App.Sce
 				{label:$L('Update'), icon:'sync', command:'sync'}
 			]};
 	this.controller.setupWidget(Mojo.Menu.commandMenu, undefined, this.cmdMenuModel);
+
+	//enable free orientation
+	this.controller.window.PalmSystem.setWindowOrientation("free");
+
   },
 
   // NOTE: this is called by app_scene's _setup function
